src/libs/Environment/betaChecker/index.android.js

```diff
@@ -1,6 +1,15 @@
 import semver from 'semver';
+import Onyx from 'react-native-onyx';
 import CONST from '../../../CONST';
 import pkg from '../../../../package.json';
+import ONYXKEYS from '../../../ONYXKEYS';
+import * as AppUpdate from '../../actions/AppUpdate';
+
+let lastBetaValue = false;
```

Comment:
This name suggests that we are storing a version string. I think we should change it to better describe that it's storing a bool:
```suggestion
let isBeta = false;
```

Reply:
@luacmartins actually its conflict with inner const isBeta = semver.gt(pkg.version, productionVersion); isBeta variable name that's why I have used it.

Reply:
Yes @0xmiroslav let me change by isLastSavedBeta

Reply:
Changes done